.github/newsletter-issue-template.md

```diff
@@ -1,15 +1,14 @@
 ---
 title: "Newsletter {{ env.NEWSLETTER_COUNTER}}: {{ env.NEWSLETTER_MONTH }} {{ env.NEWSLETTER_YEAR }}"
 labels: coordination, help wanted
-assignees: janhohenheim
+assignees: AngelOnFira, mamaicode, janhohenheim
 ---
 
-###### tags: `newsletter`
+# Newsletter {{ env.NEWSLETTER_COUNTER}}: {{ env.NEWSLETTER_MONTH }} {{ env.NEWSLETTER_YEAR }}
```

Comment:
GH issues already have a title in the, well, title. Do we need this twice?

Reply:
I don't want the PR to stay open because of a nitpick, so I'll merge this as-is.